starport/cmd/scaffold_module.go

```diff
@@ -33,6 +34,7 @@ func NewScaffoldModule() *cobra.Command {
 	}
 	c.Flags().StringSlice(flagDep, []string{}, "module dependencies (e.g. --dep account,bank)")
 	c.Flags().Bool(flagIBC, false, "scaffold an IBC module")
+	c.Flags().Bool(flagOracle, false, "scaffold a bandchain oracle module")
```

Comment:
The module will be a consumer that sends queries to the oracle isn't it?
If yes, it should be clarified in the description